### PR TITLE
Trash archive

### PR DESCRIPTION
**Pull request by devsurajrai**
I have implemented the Trash and Archive Functionality. Please Review.

https://deploy-preview-10--graceful-rugelach-

https://user-images.githubusercontent.com/77954411/166943077-1f874c62-2d65-42da-b847-014a7e7b92ab.mov

517974.netlify.app/

### CODE REVIEW

File: src/backend/controllers/NotesController.js [hunk 1]
```diff
@@ -80,7 +81,8 @@ export const deleteNoteHandler = function (schema, request) {
         }
       );
     }
-    const noteId = request.params.noteId;
+    const { noteId } = request.params;
+    console.log("noteid", noteId);
```
Comment (shraddha15-creator) [inline src/backend/controllers/NotesController.js:85]: remove consoles

File: src/backend/controllers/NotesController.js [hunk 2]
```diff
@@ -138,6 +140,8 @@ export const updateNoteHandler = function (schema, request) {
 
 export const archiveNoteHandler = function (schema, request) {
   const user = requiresAuth.call(this, request);
+  console.log("archive func", user);
```
Comment (shraddha15-creator) [inline src/backend/controllers/NotesController.js:143]: remove consoles

File: src/backendUtil/deleteRequest/deleteNotes.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import axios from "axios";
+export const deleteNotes = async (note, userToken, id) => {
+  console.log(id);
```
Comment (shraddha15-creator) [inline src/backendUtil/deleteRequest/deleteNotes.js:3]: remove

File: src/backendUtil/deleteRequest/deleteNotes.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  let config = {
+    headers: {
+      authorization: userToken,
+    },
+  };
+  try {
+    const updatedNotes = await axios.delete(`/api/notes/${id}`, config);
+    console.log("testing", updatedNotes);
```
Comment (shraddha15-creator) [inline src/backendUtil/deleteRequest/deleteNotes.js:11]: this as well

File: src/backendUtil/postRequests/postRequests.js [hunk 5]
```diff
@@ -7,7 +7,6 @@ const createNote = async (note, userToken) => {
   };
   try {
     const updatedNotes = await axios.post("/api/notes", { note: note }, config);
-    console.log(updatedNotes.data.notes);
     return updatedNotes.data.notes;
   } catch (error) {
     console.log("error creating the note", error);
```
Comment (shraddha15-creator) [inline src/backendUtil/postRequests/postRequests.js:12]: here you can use `console.error` in error handling

**Review by shraddha15-creator — APPROVED** [review]
Suggestions: 
- Check consoles and remove all
- use `console.error` while catching error

**Review by Dev-AkshatGupta — APPROVED** [review]
Everything's pretty well just go with the suggestion given by shraddha

**Pull request merged**